makefile

```diff
@@ -54,6 +54,7 @@ endif
 # For now just take the first one
 DOCKER_GOPATH_TOKS := $(subst :, ,$(DOCKER_GOPATH:v%=%))
 DOCKER_GOPATH_DIR = $(word 1,$(DOCKER_GOPATH_TOKS))
+@echo docker gopath: ${DOCKER_GOPATH_DIR}
```

Comment:
I'm unfamilar with this syntax in make.  What does this do?

Reply:
If it's not expected to be needed in the future, I'd suggest removing it.